ui/ts/models/log.ts

```diff
@@ -46,11 +54,14 @@ module Models {
       });
 
       private _url(): string {
-        let url: string = "/_status/local/log";
-        if (this.level() != null) {
-          url += "/" + this.level();
+        let url: string = "/_status/logs/";
+        if (this.node() != null) {
+          url += this.node();
         }
         url += "?";
+        if (this.level() != null) {
+          url += "level=" + this.level() + "&";
```

Comment:
@mrtracy: sadly, no, you would figure this would be something built into js
@tamird: I was already using encodeURIComponent for the pattern, but I extended this to each component just in case.